manifests/operators/monitor-cf.yml

```diff
@@ -36,24 +36,43 @@
     networks:
       - name: default
     jobs:
+      - name: bpm
+        release: bpm
       - name: firehose_exporter
         release: prometheus
+        consumes:
+          reverse_log_proxy:
+            deployment: cf
+            from: reverse_log_proxy
         properties:
           firehose_exporter:
             doppler:
               subscription_id: "((metrics_environment))"
-              max_retry_count: 300
             logging:
-              url: wss://doppler.((system_domain)):((traffic_controller_external_port))
-              use_legacy_firehose: true
-            uaa:
-              url: https://uaa.((system_domain))
-              client_id: firehose_exporter
-              client_secret: "((uaa_clients_firehose_exporter_secret))"
+              tls:
+                ca: ((firehose_exporter_to_logs_provider.ca))
+                cert: ((firehose_exporter_to_logs_provider.certificate))
+                key: ((firehose_exporter_to_logs_provider.private_key))
             metrics:
               environment: "((metrics_environment))"
             skip_ssl_verify: ((skip_ssl_verify))
 
+- type: replace
```

Comment:
Thought about this and I am wondering if that certificate creation should not be a dedicated ops file inside the CF folder, because it is kind of similar to the uaa users you need to create within CF. I know then you might need to copy it around or so, but at least the approach would be kind of consistent, or? What do you think about that

Reply:
Its quite similar but not completely equivalent. 

In case of a credential variable, the credentials needs to be deployed in UAA and therefore must appear in CF's manifest somehow.

In the case of a mTLS certificates, it won't be deployed anywhere in CF's manifest, and therefore has nothing to do in there.